glyphs-reader/src/font.rs

```diff
@@ -1862,6 +1885,31 @@ impl RawFeature {
     fn disabled(&self) -> bool {
         self.disabled == Some(1)
     }
+
+    // https://github.com/googlefonts/glyphsLib/blob/24b4d340e4c82948ba121dcfe563c1450a8e69c9/Lib/glyphsLib/builder/features.py#L134
+    fn feature_names(&self) -> String {
+        if self.labels.is_empty() {
+            String::new()
+        } else {
```

Comment:
total style nit, in a fn like this I generally prefer to use an explicit return for the guard case, so the rest of the logic isn't indented unnecessarily, so like

```rust
if self.labels.is_empty() {
    return String::new()
}
// actual logic here
```